pkg/controller/endpointslice/metrics/metrics.go

```diff
@@ -93,6 +93,29 @@ var (
 		},
 		[]string{"operation"},
 	)
+
+	// EndpointSlicesChangedPerSync observes the number of EndpointSlices
+	// changed per sync.
+	EndpointSlicesChangedPerSync = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem: EndpointSliceSubsystem,
+			Name:      "endpointslices_changed_per_sync",
+			Help:      "Number of EndpointSlices changed on each Service sync",
+		},
+		[]string{"topology"}, // either "auto" or "disabled"
+	)
+
+	// EndpointSliceSyncs tracks the number of sync operations the controller
+	// runs along with their result.
+	EndpointSliceSyncs = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      EndpointSliceSubsystem,
+			Name:           "syncs",
+			Help:           "Number of EndpointSlice syncs",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"result"}, // either "success" or "failure"
```

Comment:
this seems to have a new label now:
"success", "stale" and "failure"

Reply:
Updated, thanks!